site/_posts/2020-10-07-contour_v190.md

```diff
@@ -0,0 +1,113 @@
+https://projectcontour.io/examples/authdemo/05-proxy.yaml
+https://projectcontour.io/examples/authdemo/06-proxy-auth.yaml
+``` 
+
+## Cross-Origin Resource Sharing (CORS) Support
+Contour’s HTTPProxy API now supports specifying a [CORS policy](https://projectcontour.io/docs/main/httpproxy/#cors-policy), which configures Envoy’s [CORS filter](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/cors_filter) to allow web applications to request resources from different origins.
+
+CORS uses additional HTTP headers to tell browsers to give a web application running at one origin access to selected resources from a different origin (domain, protocol, or port) from its own.
+
+In this example, cross-domain requests will be allowed for any domain (note the * value):
+
+```yaml
+apiVersion: projectcontour.io/v1
+kind: HTTPProxy
+spec:
+  virtualhost:
+    fqdn: local.projectcontour.io
+    corsPolicy:
+        allowCredentials: true
+        allowOrigin: 
+          - "*" # allows any origin
+        allowMethods:
+          - GET
+          - POST
+          - OPTIONS
+        allowHeaders: 
+          - authorization
+          - cache-control
+        exposeHeaders: 
+          - Content-Length
+          - Content-Range
+        maxAge: "10m" # preflight requests can be cached for 10 minutes. 
+  routes:
+    - conditions:
+      - prefix: /
+      services:
+        - name: s1
+          port: 80
+```
+
+Thanks to @aberasarte and @glerchundi for driving the design and implementation of this new feature!  
+
+## Backend TLS Client Authentication
+Contour now supports optionally specifying a Kubernetes secret that Envoy should present to upstream clusters as a client certificate for TLS, so the upstream services can validate that the connection is coming from Envoy.
+ 
+Thanks to @tsaarni for leading design and implementation of this feature!
+
+## v1 Custom Resource Definitions
+Contour now generates v1 custom resource definitions (CRDs) as part of its example YAML.
+This enables Contour to take full advantage of the v1 API’s capabilities around validation, defaulting, API documentation via `kubectl explain`, and more.
+CRDs became [generally available in Kubernetes 1.16](https://kubernetes.io/blog/2019/09/18/kubernetes-1-16-release-announcement/#custom-resources-reach-general-availability) over a year ago.
+
+This change bumps Contour’s minimum supported Kubernetes version to 1.16.
+
+## Join the Contour Community
+
+* Join the [Contour Community Meetings](https://projectcontour.io/community/) or [Office Hours](https://github.com/projectcontour/community/wiki/Office-Hours)
+* Get updates on Twitter ([@projectcontour](https://twitter.com/projectcontour0))
+* Chat with us on Slack ([#contour](https://kubernetes.slack.com/messages/C8XRH2R4J) on Kubernetes Slack)
+* Collaborate with us on GitHub: [github.com/projectcontour/contour](https://github.com/projectcontour/contour)
```

Comment:
You can remove this, we have a partial layout that covers the community stuff in each blog post: https://github.com/projectcontour/contour/blob/main/site/_includes/blog-community-footer.html